contributing/code_of_conduct/concrete_example_document.rst

```diff
@@ -0,0 +1,30 @@
+Code of Conduct: Concrete Example Document
+==========================================
+
+This is a living document that serves to give concrete examples of
+unwanted behaviour. These examples have all taken place somewhere in the
+PHP community in the past, and are clear code of conduct violations
+according to the Symfony code of conduct.
+
+Concrete Examples
+-----------------
+
+- Unwelcome comments regarding a person’s lifestyle choices and practices,
+  including those related to food, health, parenting, drugs, and employment.
+- Deliberate misgendering or use of `dead names`_ (The birth name
+  of a person who has since changed their name, often a transgender person).
+- Threats of violence like "The person that created this PR should be
+  punched in the face"
+- Incitement of violence towards any individual, including encouraging a
+  person to commit suicide or to engage in self-harm
+- Sustained disruption of discussion
+- Pattern of inappropriate social contact, such as requesting/assuming
+  inappropriate levels of intimacy with others
+- Continued one-on-one communication after requests to cease
+- Putting down people based on their technology choices or their work
```

Comment:
Can you fix some minor syntatic things here:

 * Please use `*` as list indicator
 * Please end all entries with a semi-colon, except from the last item which should end with a final stop (`.`).

Reply:
Thanks! Fixed :)